lib/csrf.js

```diff
@@ -152,3 +169,17 @@ module.exports = function (options) {
         }
     };
 };
+
+function populateSet(list, startsWith, exact) {
+    list.forEach(function (config) {
+        if (typeof config === 'string') {
+            startsWith.add(config);
+        }
+        else if (config.type === 'startsWith') {
+            startsWith.add(config.path);
+        }
+        else {
+            exact.add(config.path);
```

Comment:
Should we assert if the "config" value is invalid, or "type" is unsupported (typo, case etc)?